powertools-logging/src/main/java/software/amazon/lambda/powertools/logging/internal/LambdaLoggingAspect.java

```diff
@@ -181,14 +186,24 @@ private double samplingRate(final PowertoolsLogging powertoolsLogging) {
         return args;
     }
 
+    private Optional<String> asJson(final ProceedingJoinPoint pjp,
+                                    final Object target) {
+        try {
+            return ofNullable(MAPPER.writeValueAsString(target));
+        } catch (JsonProcessingException e) {
+            logger(pjp).error("Failed logging event of type {}", target.getClass(), e);
+            return empty();
+        }
+    }
+
     private Logger logger(final ProceedingJoinPoint pjp) {
         return LogManager.getLogger(pjp.getSignature().getDeclaringType());
     }
 
     private static Optional<String> getXrayTraceId() {
         final String X_AMZN_TRACE_ID = getenv("_X_AMZN_TRACE_ID");
         if(X_AMZN_TRACE_ID != null) {
-            return ofNullable(X_AMZN_TRACE_ID.split(";")[0].replace("Root=", ""));
+            return of(X_AMZN_TRACE_ID.split(";")[0].replace("Root=", ""));
```

Comment:
Was this intentional?

Reply:
Yeah it just resolves a warning from compiler, since it was wrapping a statement which can never be null into `ofNullable `